ecs/entity.go

```diff
@@ -43,6 +43,21 @@ func (e Entity) IsZero() bool {
 	return e.id == 0
 }
 
+// ID returns the entity's ID.
+func (e Entity) ID() uint32 {
+	return uint32(e.id)
+}
+
+// Generation returns the entity's generation.
+func (e Entity) Generation() uint32 {
+	return e.gen
+}
+
+// Equals returns whether this entity is equal to another.
+func (e Entity) Equals(other Entity) bool {
```

Comment:
Is this really required? Why not use simple `==` equality?

Reply:
Good point, updated